### ISSUE TITLE
Make dedicated pages for few previous events and link in main events page

### ISSUE THREAD

**Issue by kunal-kushwaha** [WeMakeDevs/wemakedevs#567]

**Comment by hrutik7** [comment]
@Parvej2001 can I work on some components , please?

**Comment by Parvej2001** [comment]
@hrutik7 Sure . very soon we will let you know.

**Comment by hrutik7** [comment]
Yes :)

**Comment by hrutik7** [comment]
@Parvej2001 ?

**Comment by hemantwasthere** [comment]
Hii @hrutik7, I would also like to work on some components.

**Comment by hrutik7** [comment]
Need to discuss with @Parvej2001

**Comment by hemantwasthere** [comment]
Ok @hrutik7, let me know if I can contribute or not.

**Comment by Chandraprakash-Darji** [comment]
@hrutik7 @hemantwasthere let's do it, for all the page design will be same, we will follow the approach like course page. we will store content in one file and then pass that content to file and render every page.
Section can be 
- [ ] Hero - Like courses page
- [ ] Learnings - Like Meetup page Why Join section
- [ ] Glimpse of event we can add some photos, videos, or any other visual content
- [ ] Section to tell checkout ongoing events
- [ ] this sections are just in Layout component wrapper
  - [ ] Join and newsletter section  
  - [ ] FAQ
  - [ ] Partner with us 
  - [ ] Footer

I think it is fine

**Comment by Chandraprakash-Darji** [comment]
Let's create issue for particular section and work on it

**Comment by hrutik7** [comment]
Ok @Chandraprakash-Darji understood 🙂

**Comment by Lucif3r-in** [comment]
@Chandraprakash-Darji Can I too work on these issues?

**Comment by giteshsarvaiya** [comment]
Hello @Chandraprakash-Darji , I would like to work on this,  I have been following weMakeDevs since a long ago, and have a bit of knowledge about some of these events. 
I can make dedicated routes for each events with nodejs & expressJs.. and also help with reactjs and end(templates, if you may)
Kindly let me know, if openings are available and what would you want me to do.
Have a nice day!

**Comment by hemantwasthere** [comment]
@Chandraprakash-Darji Do we have to create more pages for previous events page or the issue has been solved?

**Comment by Chandraprakash-Darji** [comment]
> @hrutik7 @hemantwasthere let's do it, for all the page design will be same, we will follow the approach like course page. we will store content in one file and then pass that content to file and render every page. Section can be
> 
> * [ ]  Hero - Like courses page
> * [ ]  Learnings - Like Meetup page Why Join section
> * [ ]  Glimpse of event we can add some photos, videos, or any other visual content
> * [ ]  Section to tell checkout ongoing events
> * [ ]  this sections are just in Layout component wrapper
>   
>   * [ ]  Join and newsletter section
>   * [ ]  FAQ
>   * [ ]  Partner with us
>   * [ ]  Footer
> 
> I think it is fine

@giteshsarvaiya @hemantwasthere check this out, did you understand this?

**Comment by hemantwasthere** [comment]
> > @hrutik7 @hemantwasthere let's do it, for all the page design will be same, we will follow the approach like course page. we will store content in one file and then pass that content to file and render every page. Section can be
> > 
> > * [ ]   Hero - Like courses page
> > * [ ]   Learnings - Like Meetup page Why Join section
> > * [ ]   Glimpse of event we can add some photos, videos, or any other visual content
> > * [ ]   Section to tell checkout ongoing events
> > * [ ]   this sections are just in Layout component wrapper
> >   
> >   * [ ]   Join and newsletter section
> >   * [ ]   FAQ
> >   * [ ]   Partner with us
> >   * [ ]   Footer
> > 
> > I think it is fine
> 
> @giteshsarvaiya @hemantwasthere check this out, did you understand this?

yeah got that, but just wanted to know how many pages are there in this list which we have to create,
cuz i think most of the features have already there

**Comment by Chandraprakash-Darji** [comment]
@hemantwasthere there will one page only, we will just pass the content to it, 
like there is only on page for individual course page but whenever we visit /courses/devops then it will take content of devops and render it.

check that once, we will use same approach to it

**Comment by hemantwasthere** [comment]
Okay @Ghat0tkach, got it 👍
